packages/amplify-e2e-tests/src/__tests__/apigw.test.ts

```diff
@@ -30,7 +30,7 @@ describe('API Gateway e2e tests', () => {
     await amplifyPushAuth(projRoot);
     await addAuthWithGroupsAndAdminAPI(projRoot); // Groups: Admins, Users
     await amplifyPushAuth(projRoot);
-    await addRestApi(projRoot, { isFirstRestApi: false, path: '/foo' });
+    await addRestApi(projRoot, { isFirstRestApi: false, path: '/foo', existingLambda: true });
     await addRestApi(projRoot, { isFirstRestApi: false, restrictAccess: true, allowGuestUsers: true, hasUserPoolGroups: true });
```

Comment:
this is the line that needs to change.

Add the option `projectContainsFunctions: true`